pyxlma/lmalib/grid.py

```diff
@@ -237,9 +237,11 @@ def events_to_grid(ds, dsg, grid_spatial_coords=['grid_time',
                         keep='first').groupby('event_pixel_id')
     n_flashes = first_event_df.size()
     areas_this_pixel = first_event_df['flash_area']
+    energy_this_pixel= first_event_df['flash_energy']
     fl_mean_area = areas_this_pixel.mean()
     fl_std_area = areas_this_pixel.std()
     fl_min_area = areas_this_pixel.min()
+    fl_mean_energy=areas_this_pixel.mean()
```

Comment:
This is using the area and not the energy data.

Reply:
Thanks for catching that. I just changed it and the grids look better.